ls1intum/Artemis
# Bugfix/Re-evaluate split error

### Checklist

### Description
When re-evaluating quiz questions the method parseTextHintExplanation throws an TypeError: Cannot read property 'trim' of undefined
![image](https://user-images.githubusercontent.com/33764166/61884851-38896c80-aefd-11e9-8277-5fc8c973f00f.png)
This is caused by a wrong RegEx for splitting up the markdown text.

### Steps for Testing
1. Log in to ArTEMiS
2. Navigate to Course Administration
3. Go to a MC quiz that offers the re-evaluate button and click on it
4. Change option contents and save changes
5. Check whether the changes have been applied correctly and the browser console doesn't show the error message 

![image](https://user-images.githubusercontent.com/33764166/61896223-ccb2fe00-af14-11e9-80ec-55a5c00228b2.png)

### Screenshots
There are no UI changes
